gtsam/geometry/geometry.i

```diff
@@ -584,7 +584,13 @@ class Cal3_S2 {
 
   // Action on Point2
   gtsam::Point2 calibrate(const gtsam::Point2& p) const;
+  gtsam::Point2 calibrate(const gtsam::Point2& p,
```

Comment:
Cal3_S2 is **not** a camera, so thank you for the extra work :-) But please update PR comment (do that anyway, it's very terse) to say you also provide calibration derivatives.

PS, speaking about extra work: why not just do this in the base class? Is that because it is virtual and we can't handle?

Reply:
Most are done in the Base class, but some of them actually are different than the base class. I decided that I should not touch anything that works, so I just kept all the original and added overloads.